pepys_import/core/store/data_store.py

```diff
@@ -44,6 +46,22 @@
 DEFAULT_DATA_PATH = os.path.join(PEPYS_IMPORT_DIRECTORY, "database", "default_data")
 USER = getuser()  # Login name of the current user
 
+# Python Set Object - is this item in this set
+
+# Constant Lists of revisions - list all files in the directory
```

Comment:
on first impressions it looks like we have `postgres_versions` and `sqlite_versions` directory names the wrong way around @SamWilliams-dev